### DISCUSSION TITLE
Don't show games that are hidden on GOG

### DISCUSSION THREAD

**Discussion by Grimessa** [lutris/lutris#4901] [Ideas]
Hi

It is possible to hide games in my collection on GOG, so that they do not show up there on their website.  However Lutris shows these hidden games when loading my game collection from GOG.

I think it could be useful if Lutris does not show games in Lutris if they are hidden in GOG (Except maybe if the "Show Hidden Games" setting is checked in Lutris).  

A use case of this would be that I do not want all my games to show up on the family computer that my kids use.

Regards

**Comment by strycore** [comment]
It may be possible if the GOG API exposes this hidden setting

**Comment by strycore** [comment]
```{"isGalaxyCompatible": true, "tags": [], "id": 1, "availability": {"isAvailable": false, "isAvailableInAccount": true}, "title": "Fallout Classic", "image": "//images-1.gog-statics.com/0ef80c3dc9692710809e4adaf2fcd3348f9ede1f530af6182c2972c86238e71a", "url": "", "worksOn": {"Windows": true, "Mac": true, "Linux": false}, "category": "Role-playing", "rating": 48, "isComingSoon": false, "isMovie": false, "isGame": true, "slug": "fallout_classic", "updates": 0, "isNew": false, "dlcCount": 0, "releaseDate": {"date": "1997-09-30 00:00:00.000000", "timezone_type": 3, "timezone": "Europe/Nicosia"}, "isBaseProductMissing": false, "isHidingDisabled": false, "isInDevelopment": false, "extraInfo": [], "isHidden": false}```


There's a isHidden parameter. Now the problem is that the hidden field in Lutris affect installed games, not the libraries from various stores. But that can be arranged.

**Reply by Grimessa** [reply]
That is great, thanks for checking their API.

To me it would make sense if the hidden field controls both installed games and stores.  That seems intuitive to me.🤔  Though I am no expert on Lutris' UI.

**Reply by strycore** [reply]
It has more to do with the database than the UI, but yes, I think it should make sense to have both
